README.md

```diff
@@ -180,6 +180,8 @@ For CoreOS, use:
 
 `-start`: Run waagent as a background process
 
+`-collect-logs [-full]`: Runs the log collector utility that collects relevant agent logs for debugging and stores them in the agent folder on disk. Exact location will be shown when run. Use flag `-full` for more exhaustive log collection. 
```

Comment:
>stores them in the agent folder on disk

Would it be better to specify the exact directory here?

Reply:
It's configurable.